c2rust-ast-exporter/src/AstExporter.cpp

```diff
@@ -2213,8 +2213,12 @@ class TranslateASTVisitor final
 
                          // 2. Encode bitfield width if any
                          if (D->isBitField()) {
+#if LLVM_VERSION_MAJOR >= 20
+                             cbor_encode_uint(array, D->getBitWidthValue());
```

Comment:
One little nit: could you use a bit width variable inside the `#if` and then a single `cbor_encode_uint` call outside of it?

Reply:
Sorry but I'm not familiar with code here and not sure about how to name this variable (`bitWidth` is already occupied in this scope)

Reply:
Okay, I did it here on your PR (bc656a7).